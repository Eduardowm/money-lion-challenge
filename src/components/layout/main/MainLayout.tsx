import { PropsWithChildren } from 'react'

function MainLayout({ children }: PropsWithChildren) {
  return <main className="flex min-h-screen flex-col bg-gray-900 text-white">{children}</main>
}
export default MainLayout
