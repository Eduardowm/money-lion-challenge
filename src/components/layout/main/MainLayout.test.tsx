import { render } from '@testing-library/react'

import MainLayout from './MainLayout'

describe('MainLayout', () => {
  it('renders as expected', () => {
    const { container } = render(<MainLayout>This is the main layout content</MainLayout>)

    expect(container).toMatchSnapshot()
  })
})
