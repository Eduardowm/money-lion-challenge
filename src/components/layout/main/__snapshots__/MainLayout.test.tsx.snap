// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout renders as expected 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col bg-gray-900 text-white"
  >
    This is the main layout content
  </main>
</div>
`;
